pkg/configurator/methods.go

```diff
@@ -16,6 +16,15 @@ import (
 const (
 	// defaultServiceCertValidityDuration is the default validity duration for service certificates
 	defaultServiceCertValidityDuration = 24 * time.Hour
+
+	// defaultCertKeyBitSize is the default certificate key bit size
+	defaultCertKeyBitSize = 2048
+
+	// minCertKeyBitSize is the minimum certificate key bit size
+	minCertKeyBitSize = 512
```

Comment:
I'd set the min to `2048`. Decreasing this to 512 allows for a much lower key strength, which I don't see a use case for atm.